### ISSUE TITLE
[FEATURE] Display date/time on the event list for each event

### ISSUE THREAD

**Issue by eddiejaoude** [EddieHubCommunity/EddieHubLive#41]
### Description

It would be good to understand the date and time of the event in the event list

**Comment by neerajram30** [comment]
I was working on it, but can't fetch data from endpoint.

**Comment by neerajram30** [comment]
Thanks for helping out. It is now working. I need to know that in which format the date and time has to printed in event list.

**Comment by RohanHarichandan** [comment]
Hey @eddiejaoude I'm also facing the same problem no data is showing, can you please explain what you actually mean " then edit your message above and the event will be displayed " in this statement

**Comment by realArafatBen** [comment]
Hi, have the issue been resolved?

**Comment by RohitRajvaidya5** [comment]
Is this issue resolved I want to contribute

**Comment by eddiejaoude** [comment]
It doesn't look like this issue has been resolved. 

- the date format can be something friendly but should include time (this can always be tweaked later)
- to get data, get the app running locally and you will see no events, then edit your message above and the event will be displayed - you can keep editing your message, that way you get an event but it won't notify everyone

**Comment by RohitRajvaidya5** [comment]
I am sorry I don't understand "edit message above" is it message above your comment ?
Because I try that and doesn't work?

**Comment by jalezi** [comment]
If I edit this message, I should see the event locally?

**Comment by Favalcodes** [comment]
has this issue been done? @eddiejaoude if not, can I be assigned to it?

**Comment by Nymphalys** [comment]
Hi I would Like tto work on this issue,I think I can fix this

**Comment by Vishrut19** [comment]
Hi I would like to contribute to this issue @eddiejaoude

**Comment by eddiejaoude** [comment]
If you edit your message in the issue, the event will be sent to our `api` and then to your locally running `live` map app, so you will get data

@neerajram30 sorry for the slow reply, do you still want the issue?

**Comment by amandamartin-dev** [comment]
@eddiejaoude is this issue claimed?  I love anything with a map and could take this or something else on in here potentially! I did click on the prod link and didn't see any data returned though, but this could be because I haven't interacted with the issues in this repo yet.  Let me know!

**Comment by eddiejaoude** [comment]
Sure Amanda, I can assign this to you 👍

**Comment by amandamartin-dev** [comment]
@eddiejaoude this is interesting! I actually just submitted a bug as I was not seeing any data live when I would interact with this project! I do see yours now but not mine.  Shall I leave that bug open or am I misunderstanding the origin? https://github.com/EddieHubCommunity/EddieHubLive/issues/51

**Comment by eddiejaoude** [comment]
I think your issue is correct, because we need to improve the docs, so that people know how to enable their data to be displayed on the map project

**Comment by amandamartin-dev** [comment]
I closed the bug, making a docs specific issue now

**Comment by amandamartin-dev** [comment]
I have this running locally now and a few thoughts/questions:

1. Do we really need the date or just the time?  I noticed that in the current version of this site, it seems to only show data from the same day, so it seems date is not necessarily needed

3. I am intending to use 24 hour format to display the time so it is more universal and also display based on the client location.  If there are any reasons not to do this, lmk!

**Comment by eddiejaoude** [comment]
Thanks Amanda, great points 👍 

1. I think the `time` is enough for now, because you are right, we are mostly interested in the recent events
2. oh great idea 🔥 

I think a filter could come later, having a limit makes sense, see what makes sense and we can always tweak later 👍 

Sounds great!

**Comment by amandamartin-dev** [comment]
Before I finish this up, adding screenshots for a UI check. I put the time in the upper right hand corner of the cards with no label, but if it's not obvious enough could add "updated at" or something.  Just didn't want to clutter if it's clear enough this way

East Coast US test
![us_test](https://user-images.githubusercontent.com/97615019/203999755-fd97584c-128f-4e66-852e-2ad488720b02.jpg)

UK test
![uk_test](https://user-images.githubusercontent.com/97615019/203999881-c60870cc-d996-44cc-9f6e-fe749288977a.jpg)

I also know @Cahllagerfeld is working on UI changes to the cards in another ticket, so thinking I should hold on this and pull in his changes first once complete?  Let me know what the preferred process is here.

**Comment by Cahllagerfeld** [comment]
Nice work!
If the change to the card suggested in #55 is okay, we can ship it. This should unblock this issue.

**Comment by amandamartin-dev** [comment]
It looks good to me, I haven't reviewed a PR here yet but I'll pop over and move it along if that's okay
